sbt/common.sbt

```
@@ -0,0 +1,38 @@
+// Copyright (C) from 2022 The Play Framework Contributors <https://github.com/playframework>, 2011-2021 Lightbend Inc. <https://www.lightbend.com>
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// NOTE: !!! THIS IS A COPY !!!                                                                                //
+// To edit this file use the main version in https://github.com/playframework/.github/blob/main/sbt/common.sbt //
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * If you need an extra commands to format source code, add the next code to your `build.sbt`
+ * {{{
+ *   val _ = sys.props += ("sbt_formatCode" -> List(...).mkString(";"))
```

My suggestion (because you could format any document):
```suggestion
 * If you need extra commands to format source code or other documents, add following line to your `build.sbt`
 * {{{
 *   val _ = sys.props += ("sbt_formatCode" -> List("<command1>", "<command2>",...).mkString(";"))
```

---

Done ✔️